vertical-pod-autoscaler/docs/examples.md

```diff
@@ -108,3 +109,16 @@ These options cannot be used together and are mutually exclusive.
 It is possible to set the failurePolicy of the webhook to `Fail` by passing `--webhook-failure-policy-fail=true` to the VPA admission controller.
 Please use this option with caution as it may be possible to break Pod creation if there is a failure with the VPA.
 Using it in conjunction with `--ignored-vpa-object-namespaces=kube-system` or `--vpa-object-namespace` to reduce risk.
+
+### Specifying global maximum allowed resources to prevent pods from being unschedulable
```

Comment:
What do you think about moving this section move to the "features.md" page?

Reply:
There are many sections in examples.md which actually describe features of VPA (Starting multiple recommenders, Custom memory bump-up after OOMKill, Using CPU management with static policy, Controlling eviction behavior based on scaling direction and resource, etc.). I don't think the newly introduced section is different from the existing section. 
IMO, examples and features are overlapping conceptually. If you describe a feature, you usually also add example(s) of how the feature can be used. `examples.md` and `features.md` should be merged IMO. This is out-of-scope of the existing PR.

Reply:
> IMO, examples and features are overlapping conceptually. If you describe a feature, you usually also add example(s) of how the feature can be used. `examples.md` and `features.md` should be merged IMO. This is out-of-scope of the existing PR.

I agree with you here. The features page is new, and I want to start moving the examples across to the features page, with more description about that feature. 

I thought it would be nice for the "global max" feature to be added to features from the start, since it's a better fit there, and will require work to move at a later stage.

Reply:
(but it's up to you though, the documentation needs a lot of work in general)

Reply:
I would prefer to keep it consistent with the existing doc. In another PR all the examples doc can be reworked as features sections.